src/client/driver/index.js

```diff
@@ -10,6 +10,7 @@ import * as browser from '../browser';
 import executeActionCommand from './command-executors/execute-action-command';
 import executeWaitForElementCommand from './command-executors/execute-wait-for-element-command';
 import executeNavigateToCommand from './command-executors/execute-navigate-to-command';
+import executePendingBrowserWindowCommand from './command-executors/execute-pending-browser-window-command';
```

Comment:
So let's get rid of something else...
Do we really need the `...PendingBrowser...` thing here?
We have no other window than browser's. And... is it the possible task's lifecycle: it becomes pending and then gets executed?
If yes for both, let's stick to:

`RunWindowTaskCommand`

and as to the function that triggers it - if it's *function* not *command* can it be just `ExecuteWindowTask`? By triggering the respective command, we imply.

Did i get rid of too much?